BuildROS2/setup_ros2_base.sh

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env bash
 
 UE_PROJ_PATH=${1:-~/UnrealEngine}
-CLEANUP=${2:-"false"}
```

Comment:
the second argument is never used in this script?

Reply:
setup_ros2_base.sh and setup_ros2_pkg.sh are used by same python and pkgs.sh has $2
need some more refactor to fix this to support build without cleanup